tsconfig.sdk.json

```diff
@@ -0,0 +1,38 @@
+{
+  "extends": "./tsconfig.base.json",
+  "compilerOptions": {
+    "declaration": true,
+    "emitDeclarationOnly": true,
+    "outDir": "./resources/embedding-sdk/dist",
+    "paths": {
+      "*": [
+        "./frontend/src/*",
+        "./frontend/test/*",
+        "./enterprise/frontend/src/*",
+        "./enterprise/frontend/test/*"
+      ],
+      "cljs/*": ["./target/cljs_dev/*"]
+    }
+  },
+  "include": [
+    "frontend/src/**/*",
+    "enterprise/frontend/src/**/*",
+    "frontend/test/**/*",
+    "enterprise/frontend/test/**/*"
+  ],
+  "exclude": [
+    "node_modules",
+    // The following files will load `frontend/src/metabase/app` which will conflict with
+    // `frontend/src/metabase/App.tsx`, since now we're loading both JS and TS files.
```

Comment:
We probably should do something about this (not in this PR though). VS Code confuses `frontend/src/metabase/App.tsx` with `frontend/src/metabase/app.js` and that's frustrated when working with these files.

Reply:
maybe renaming `app.js` to `init.js` could help, as it seems it's just code to init the app

Reply:
yeah, that would do. We don't have `init.*` in `frontend/src/metabase/` currently.

Reply:
Yeah, I know that because I wrote that comment 😅 . It's just that this is something we should be aware of at least because I'm not particularly satisfied with the solution I went with either.